packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -247,6 +257,36 @@ export const Link = styled.a`
   }
 `;
 
+const StyledLiveLabel = styled.span`
+  color: ${C_POSTBOX};
+  ${({ service }) => getSansBold(service)}
+  text-decoration:none;
+  display: inline-block;
+  ${({ dir }) =>
+    dir === 'rtl' ? 'margin-left: 0.5rem;' : 'margin-right: 0.5rem;'}
+`;
+
+export const LiveLabel = ({ service, text, dir }) => (
+  // eslint-disable-next-line jsx-a11y/aria-role
+  <span role="text">
+    <StyledLiveLabel service={service} aria-hidden="true" dir={dir}>
+      LIVE
```

Comment:
Guess we don't care about this hardcoded english yet?

Reply:
We can add a `label` prop instead of hardcoding, so that we would be able to pass a translation if needed.

Reply:
Sounds good :D

Reply:
Or alternatively we could update it to receive children (`text node`)

Reply:
Good point - `<LiveLabel>Live</LiveLabel>` would probably be clearer than `<LiveLabel label="Live" />` 👍

Reply:
I think this will always be in English... unless it's a RTL language